pkg/api/v1/pod/util.go

```diff
@@ -24,6 +24,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/apis/scheduling"
```

Comment:
I wouldn't expect to add references to internal APIs packages here. This helper package is dealing with external APIs, so ideally we would only use things under `k8s.io/api/...`. Should the `DefaultPriorityWhenNoDefaultClassExists` constant be placed in the external API package as well?

Reply:
The only other place it is used is in the admission plugin. If moving it to external API package keeps the admission plugin logic intact we can go for it. Should I keep it in this PR as a commit or open a separate PR for the same?

Reply:
Will open a separate PR for same to keep this PR relevant.

Reply:
this should be resolved before merging this PR